File: java/8.4/example1.java
import java.util.Scanner;

public class example1 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a colour");
        String colour = scanner.nextLine();
        System.out.println("Enter your name");
        String name = scanner.nextLine();
        System.out.println(name + " Your favourite colour is " + colour);
    }
}

File: rust/src/e84/example1.rs
pub fn main() {
    let mut color = String::new();
    let mut name = String::new();
    println!("Enter a colour");
    std::io::stdin().read_line(&mut color).unwrap();
    println!("Enter a name");
    std::io::stdin().read_line(&mut name).unwrap();
    println!("{} your faviroute color is {}.", name.trim(), color.trim());
}